src/Components/Playlist/PlaylistContainer.js

```diff
@@ -0,0 +1,87 @@
+    const [ isError, setIsError] = useState('')
+    const addToPlaylist = () =>{
+        if(title===''){
+            setIsError('This field cannot be empty')
+        }
+        else{
+            addPlaylist(title, description)
+            setIsError('')
+            setDescription('')
+            setTitle('')
+        }
+        
+    }
+    return(
+        <div class="playlist-modal">
+            <div class="modal-body">
+                <div className="modal-header">
+                    {playlist!==undefined && playlist.length>0 ?  
+                        <span className="mb-1">save to Playlist</span>
+                    :
+                    <span className="mb-1">Create new Playlist</span>
+                    }
+                    <span><i className="bi bi-x" onClick={() => {
+                        setPlaylistModal(false)}}></i></span>
+                </div>
+                <div className="playlist-title">
+                    {playlist&&
+                        playlist.map((item) => 
+                            <div className="playlist-content">
+                                {item.videos.some((v) => v.id===video.id) 
+                                ? 
+                                <input 
+                                    className="checkbox-input"
+                                    type="checkbox"
+                                    checked={item.videos.some((v) => v.id===video.id) ? true : false}
+                                    onChange={() => {
+                                        deleteSinglePlaylist(item._id, video._id, setPlaylist)
+                                    }}/>
+                                :
+                                <input 
+                                    className="checkbox-input"
+                                    type="checkbox"    
+                                    checked={item.videos.some((v) => v.id===video.id) ? true : false}
+                                    onChange={() =>{
+                                        addToSinglePlaylist( item._id, video, setPlaylist)
+                                    }}/>
+                                }
+                                <span>{item.title}</span>
+                            </div>
+                        )
+                    }
+                </div>
+            <input 
+                type="text"
+                className="title"
+                placeholder="Enter title"
+                value={title}
+                onChange={(e) =>setTitle(e.target.value)}>
+            </input>
+            <input 
+                placeholder="Enter description"
+                className="description"
+                value={description}
+                onChange={(e) =>setDescription(e.target.value)}>
+            </input>
+            <div>
+            <div className="text-danger">{isError}</div>
```

Comment:
Suggestion: you 'isState' for boolean value like 'isError', 'isClosed' etc. I don't think 'isError' here is a boolean value.